lib/codebreaker_console/user_creation.rb

```diff
@@ -0,0 +1,52 @@
+module CodebreakerConsole
+  class UserCreation
+    include View
+
+    def initialize(game)
+      @game = game
+    end
+
+    def create_user
+      name = name_init
+      difficulty = difficulty_init
+      case difficulty
+      when 'easy' then NewSuperCodebreaker2021::User.new(name, 0)
+      when 'medium' then NewSuperCodebreaker2021::User.new(name, 1)
+      when 'hell' then NewSuperCodebreaker2021::User.new(name, 2)
```

Comment:
Why we use ints instead `easy`, `medium`, `hell`  inside gem?

Reply:
Because it is easier to sort the difficulties this way and their names may change and there will be no need to change the entire database, but only view